client/state/startup-flags/actions.ts

```diff
@@ -0,0 +1,20 @@
+import { warningNotice } from 'calypso/state/notices/actions';
+import { STARTUP_STATES_RANDOM_CLEAR_STATE_NOTIFIED } from '../action-types';
+
+export function developerNotificationCompleted() {
```

Comment:
Using Redux for this seems a bit over-engineered to me. Wouldn't it be simpler if we use a flag inside `persisted-state` and updating that flag when persisted state is cleared? Then using a simple selector-like function that returns the flag?

Reply:
I didn't want to use the browser store because we don't need the flag to be persistent. And I was modelling this state slice as a place to maintain any future such flags. But if you think it's better to just move it to persistent storage I can go ahead with it.

Reply:
I'm not saying to persist to the browser; but just to have a simple flag in the `persisted-state` module. That flag would indicate whether the user was notified, and would default fo `false`, then we'd set it to `true` inside the component when we dispatch the notice. The behavior is simple enough and we want to do it just once, so that flag should do the job. I'm really hesitant to use Redux and specifically add a new key that loads for everyone for such a simple "toggle-like" behavior.

Reply:
Got it @tyxla let me give it a try!

Reply:
I have removed any redux involvement!